### ISSUE TITLE
Blocking the "received" tick for messages

### ISSUE THREAD

**Issue by freakpopular** [tomer8007/whatsapp-web-incognito#88]
I would like to have the possibility of receiving messages blocking the second tick of the message, that means, showing to the sender that I haven't received the message. Is it possible?

**Comment by tomer8007** [comment]
Yes, this should be possible and even easy to do, but it doesn't sound like a very popular feature that is worth the GUI space.

**Comment by tomer8007** [comment]
@freakpopular  I think the main problem with this feature is that it'll easy be confusing if your phone is online and will send the tick instead..
